src/python-fastui/tests/test_forms.py

```diff
@@ -6,14 +6,14 @@
 from fastapi import HTTPException
 from fastui import components
 from fastui.forms import FormFile, fastui_form
-from pydantic import BaseModel
+from pydantic import BaseModel, Field
 from starlette.datastructures import FormData, Headers, UploadFile
 from typing_extensions import Annotated
 
 
 class SimpleForm(BaseModel):
-    name: str
-    size: int = 4
+    name: str = Field(..., max_length=10, min_length=2, description='This field is required, it must have length 2-10')
```

Comment:
the simple model should stay simple, instead of changing this, please create a new model with constraints.